src/com/insightdatascience/onpoint/lib/amazonqasampler.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# coding: utf-8
+# Author: Hairong Wang
+
+import json
+import pandas as pd
+
+# Complete file name for amazonqa training dataset(squad format)
+INFILE = "path/to/input/file"
+OUTFILE = "path/to/output/file"
+
+class AmazonQASampler:
+    def __init__(self, infile):
+        self._infile = infile
+
+    def getDF(self):
```

Comment:
Change "getDF" to "get_df". Also need to change all the other corresponding ones in the package. Here is the sytle guide: 
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

> Function and Variable Names

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.

> Variable names follow the same convention as function names.

> mixedCase is allowed only in contexts where that's already the prevailing style (e.g. threading.py), to retain backwards compatibility.

Reply:
Done, thanks for the guide!